crypto/cipher/aes_icm.c

```diff
@@ -291,10 +293,11 @@ static void srtp_aes_icm_advance (srtp_aes_icm_ctx_t *c)
  *  - fill buffer then add in remaining (< 16) bytes of keystream
  */
 
-static srtp_err_status_t srtp_aes_icm_encrypt (void *cv,
-                                               unsigned char *buf, unsigned int *enc_len)
+static srtp_err_status_t srtp_aes_icm_encrypt(void *cv,
```

Comment:
Notices an "e" out of place a few lines above (line 283 in the new code).  That's not from these changes, but ... wanna fix it?

Reply:
done